gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/JWTMessages.java

```diff
@@ -59,8 +59,11 @@
   @Message( level = MessageLevel.DEBUG, text = "Audience claim has been validated." )
   void jwtAudienceValidated();
 
-  @Message( level = MessageLevel.INFO, text = "Path {0} is configured as unauthenticated path, letting the request {1} through" )
-  void unauthenticatedPathBypass(String path, String uri);
+  @Message( level = MessageLevel.INFO, text = "Request {0} matches unauthenticated path list {1}, letting it through" )
+  void unauthenticatedPathBypass(String uri, String unauthPathList);
+
+  @Message( level = MessageLevel.ERROR, text = "Error {1} while checking whether path {0} should be allowed unauthenticated access" )
```

Comment:
nit: I would prefer to see the error at the end of the message; I think it's clearer that way.
"Error while checking whether path {0} should be allowed unauthenticated access : {1}"

Reply:
+1